### ISSUE TITLE
Error occur when clicking on the "Connect to OneDrive.com" button

### ISSUE THREAD

**Issue by evergreen007** [xybu/onedrive-d-old#40]
The installation seems right, till the end, when the "preferences" window appear, I clicked on the "Connect to the OneDrive.com" buttion:
An error occured:

:~$ onedrive-prefs NOT SANDBOXED [ERROR:flash/platform/pepper/common/pep_process.cpp(19)] CHECK(g_process_type == PROCESS_TYPE_UNKNOWN) failed java version "1.7.0_55" OpenJDK Runtime Environment (IcedTea 2.4.7) (7u55-2.4.7-1ubuntu1) OpenJDK 64-Bit Server VM (build 24.51-b03, mixed mode) Aborted (core dumped)

Platform: Ubuntu 14.04LTS

**Comment by xybu** [comment]
I am sorry but could you describe more about the environment you are running the program? I have no idea why Java gets involved.

**Comment by evergreen007** [comment]
I'm sorry for so late reply. My platform: ubuntu 14.04LTS.
I followed the instruction: http://www.omgubuntu.co.uk/2014/06/one-drive-ubuntu-integration?utm_source=feedly&utm_reader=feedly&utm_medium=rss&utm_campaign=one-drive-ubuntu-integration
However, after a while till today, I've just tried again and everything's fine, It's running.
I think, there's a difference here, I installed the java7 by following from the instruction: http://www.webupd8.org/2014/04/oracle-java-installer-conflicting-with.html
Thank you for interesting.
